neuromation/api/images.py

```diff
@@ -172,19 +171,18 @@ def _validate_image_for_tags(self, image: RemoteImage) -> None:
         if not image.name:
             raise ValueError(err + "missing image name")
 
-    async def tags(self, image: RemoteImage) -> List[RemoteImage]:
+    async def tags(self, image: RemoteImage) -> List[str]:
```

Comment:
If you are touching the method -- please convert it into async generator that yields strings (if you prefer it over RemoteImage, I have no strong opinion here).

Please apply the same transformation to `images.ls()`